compiler/damlc/daml-stdlib-src/DA/Bifunctor.daml

```diff
@@ -7,107 +7,107 @@ module DA.Bifunctor(Bifunctor(..)) where
 
 -- | A bifunctor is a type constructor that takes
 -- two type arguments and is a functor in /both/ arguments. That
--- is, unlike with 'Functor', a type constructor such as 'Either'
--- does not need to be partially applied for a 'Bifunctor'
+-- is, unlike with Functor, a type constructor such as Either
```

Comment:
Don't we want `Functor` to be typeset as code anymore?

Reply:
That is not what apostrophes do in our docs. They are just rendered as text. But I'll replace them with backticks where appropriate so they become inline literals.

Reply:
In Haddock proper, apostrophes typeset text as code and back it with a link to the definition of the identifier. I'm not sure we can do the latter but we should definitely do the former by using backticks. Thanks for taking care of it.